tests/commands/test_store_api.py

```diff
@@ -78,6 +90,27 @@ def test_client_init_ephemeral(config):
     ]
 
 
+# -- tests for anonymized client usage
+
+
+def test_anonymized_client_init(config):
+    """Check that the client is initiated ok even without config."""
+    with patch("charmcraft.commands.store.store.AnonymizedClient") as anonymized_client_mock:
```

Comment:
is this mock necessary? What happens if you don't mock it and instead just check the `_client` attribute of the created store?

Reply:
That will also works, but I reused exists test case from the normal client.